src/config.h

```diff
@@ -63,7 +63,7 @@ class Config {
   uint64_t max_probes = 1024;
   uint64_t max_strlen = 1024;
   uint64_t on_stack_limit = 32;
-  uint64_t perf_rb_pages = 64;
+  uint64_t perf_rb_pages = 4096;
```

Comment:
I know I suggested this size, but I'm just thinking what about resource constrained devices? Maybe we could just have a util that reads `/proc/meminfo` and scales down suitably, e.g. `256 * std::min(system_memory_in_mb/64, 16)`;

Reply:
Yes. I was thinking of doing this weeks ago when I planned on making this change but forgot. Will update.